py/desispec/io/util.py

```diff
@@ -745,7 +749,7 @@ def camword_to_spectros(camword, full_spectros_only=False):
             spectros.add(int(char))
         elif full_spectros_only and char in ['b','r','z']:
             break
-    return list(spectros)
+    return sorted(spectros)
```

Comment:
I think there is a bug here that pre-dates this PR for non-standard but valid camwords like 'b0a12':
```
camword_to_spectros('b0a12', full_spectros_only=True)
--> [] instead of [1,2]
```

I think we want to standardize the camword input with something like
```
camword = parse_cameras(decode_camword(camword), loglevel='error')
```
before proceeding with the rest of the logic.

Let's also add a comment in the docstring that this returns a sorted list of integer spectrograph numbers..., i.e. it could potentially be in a different order than they appear in the camword if an unusual non-sorted camword is supplied.

Reply:
The "standard" (admittedly not well documented, if at all) for camwords is that they are in alphanumeric order, which your bug example isn't. I don't want to support every arbitrary camword, e.g. `z9a543b1r2`, but your workaround is simple and shouldn't be too compute-heavy so I am happy to implement it here. This code can be run tens of thousands of times in e.g. the dashboard code, which is why I didn't originally include all these set manipulations to standardize it in the initial implementation.